src/nls/de/strings.json

```diff
@@ -52,44 +52,48 @@
         "NEW_VECTOR_MASK_FROM_SHAPE": "Vektormaske aus Formebene erstellen"
     },
     "APP_NAME": "Photoshop",
+    "GO_URL": {
+        "EXPERIENCE_DESIGN": "[9047748] ü_http://www.adobe.com/go/experience-design",
+        "DESIGNSPACE_HELP": "[9047747] ü_http://www.adobe.com/go/designspace-help"
+    },
     "FIRST_LAUNCH": {
         "CONTINUE": "Fortsetzen",
         "GET_STARTED": "Erste Schritte",
-        "SLIDES": {
-            "0": {
```

Comment:
We need to preserve the structural change here, using a number-indexed object instead of an array.